src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -137,7 +137,6 @@ public static void DigestAuthentication_RoundTrips_Echo()
 
     [Fact]
     [OuterLoop]
-    [ActiveIssue(604, PlatformID.AnyUnix)]
     public static void NtlmAuthentication_RoundTrips_Echo()
```

Comment:
Those tests are using default credentials. We should have a test which uses explicit credentials with NTLM. I can't remember whether NTLM requires a domain or whether you can authenticate against a local machine account. If the later, then we could have NTLM tests running regularly.